package.json

```diff
@@ -56,6 +53,7 @@
     "webpack": "^3.3.0"
   },
   "dependencies": {
+    "aws-sdk": "^2.390.0",
```

Comment:
do we need to bundle aws-sdk for lambda? I believe it should be globally available in AWS environment.

Reply:
true